common/src/main/java/com/market/banica/common/util/ApplicationDirectory.java

```diff
@@ -0,0 +1,33 @@
+package com.market.banica.common.util;
+
+import java.io.File;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+
+public class ApplicationDirectory {
```

Comment:
You can name it ApplicationDirectoryUtil

Reply:
Renamed it and updated called instances.